build/manifest/main.go

```diff
@@ -10,7 +10,11 @@ import (
 	"github.com/pkg/errors"
 )
 
-const pluginIDGoFileTemplate = `package main
+const autogeneratedFileComment = "// This file is automatically generated. Do not modify it manually."
```

Comment:
It would be duplicated, but I'm could we just embed this comment into the `pluginIDGoFileTemplate` directly? I think it would read more cleanly.